Woof/Woof/View/SitterListView/SitterListView.swift

```diff
@@ -0,0 +1,21 @@
+import SwiftUI
+
+/// A view displaying the list of available sitters.
```

Comment:
nit:

Are we going to display only available sitters?

Reply:
Fixed. Maybe it's better to not specify what sitters will be displayed.